architecture/service_catalog/index.adoc

```diff
@@ -185,15 +204,44 @@ in which pods.
 +
 .Example `Binding` Resource
 ----
-apiVersion: servicecatalog.k8s.io/v1alpha1
-kind: Binding
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceBinding
 metadata:
   name: myBinding
+  namespace: test-ns
 spec:
+  instanceRef:
+    name: my_db
+  parameters:
+    securityLevel: confidential
   secretName: mySecret
-  <pod_selector_labels>
 ----
 
+Parameters::
+A _parameter_ is a special field available to pass additional data to the broker 
+when using either bindings or instances. The only formatting requirement is for
+the parameters to be valid YAML (or JSON). In the above example, a security
+level parameter is passed to the broker in the binding request. For parameters
+that need more security, the parameter(s) should be placed in a secret and then
```

Comment:
For parameters that need more security, place them in a secret and reference them using `parametersFrom`.